src/main/java/executor/command/CommandConvert.java

```diff
@@ -0,0 +1,194 @@
+package executor.command;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.math.RoundingMode;
+import java.net.URL;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.Ui;
+import ui.Wallet;
+import java.math.BigDecimal;
+
+/**
+ * This command helps to convert user entered currency amount into another country's currency amount using api call.
+ */
+public class CommandConvert extends Command {
+    private String from = " ";
+    private String to = " ";
+    private Double amount;
+    private Double exchangeRate = 0.00;
+
+    /**
+     * Constructor for the CommandConvert class.
+     * @param userInput this is the userInput from the CLI
+     */
+    public CommandConvert(String userInput) {
+        this.userInput = userInput;
+        this.commandType = CommandType.CONVERT;
+        this.amount = extractAmount(this.commandType, userInput);
+        this.from = getFromUserInput(userInput);
+        this.to = getToUserInput(userInput);
+    }
+
+
+    /**
+     * execute is the overall function which calls the converter and ui.
+     * @param wallet wallet is the overall wallet object
+     */
```

Comment:
Doesn't require a JavaDoc imo but if you want to keep it, describe it better i.e. Runs this Command that converts the user input cash amount from one currency to another and print it in the User Interface.

Under @param, the phrase 'wallet object' should be capital 'W' because you're talking about the Class not the object. Actually it's very weird to have a param passed in but not used. Just take note in the future maybe we can improve it.

Reply:
Removed the javadoc but updated the sample comment as per suggestion proposed.